src/commands/runDraftTool/runDraftSetupGH.ts

```diff
@@ -7,7 +7,7 @@ import { runDraftCommand } from './helper/runDraftHelper';
 import { reporter } from './../../utils/reporter';
 import { MultiStepInput, shouldResume } from './model/multiStep';
 
-export default async function runDraftCreateCmdPalette(
+export default async function runDraftSetupGH(
```

Comment:
💡 just fyi - there is a create function and then other is setupGH, so not sure what other refactors have done but naming convention varied previously because of those reasons.